packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -86,3 +88,37 @@ test('stakeBld', async t => {
     },
   });
 });
+
+test('stakeAtom', async t => {
```

Comment:
this is a good start. can you get it to interact with the smart wallet using mocked bridge responses?

happy to pair on that